drupal-org.make

```diff
@@ -53,19 +53,22 @@ projects[libraries][version] = 3.0-alpha1
 projects[lightning_api][type] = module
 projects[lightning_api][version] = 4.4
 projects[lightning_core][type] = module
-projects[lightning_core][version] = 5.0
+projects[lightning_core][download][type] = git
```

Comment:
Something doesn't make sense here. Why did this change? I'm surprised that it would have.